app/scripts/lib/rpc-method-middleware/handlers/log-web3-usage.js

```diff
@@ -45,10 +45,11 @@ function logWeb3UsageHandler (
   }
   if (!recordedWeb3Usage[origin][name]) {
     recordedWeb3Usage[origin][name] = true
+
     sendMetrics({
-      action,
-      name,
-      customVariables: { origin },
+      event: `window.web3 Usage`,
```

Comment:
Events should be in proper case and past tense. `Dapp Interacted With window.web3`

Reply:
64bd491